### PR TITLE
[MT-4785] Option for changing visibility of Open in new tab

### PR DESCRIPTION
**Pull request by OskarGasiorowski**

### CODE REVIEW

File: packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/LinkMenu.tsx [hunk 1]
```diff
@@ -11,13 +11,22 @@ import { STRING_URL_PATTERN } from '#modules/editor-v4-components/LinkMenu';
 interface Props {
     node: LinkNode | null;
     canUnlink: boolean;
+    showOpenInNewTabOption: boolean;
```
Comment (e1himself) [inline packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/LinkMenu.tsx:14]: I suggest unifying the property name. 

Right now the same thing has three different names in different components:

- `linksOpenInNewTabOption`
- `showOpenInNewTabOption`
- `withLinksOpenInNewTabOption`

How about renaming it to `withNewTabOption` everywhere?

File: packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/LinkMenu.tsx [hunk 2]
```diff
@@ -11,13 +11,22 @@ import { STRING_URL_PATTERN } from '#modules/editor-v4-components/LinkMenu';
...
     onBlur: () => void;
     onChange: (props: Pick<LinkNode, 'href' | 'new_tab'>) => void;
     onClose: () => void;
     onUnlink: () => void;
 }
 
-export function LinkMenu({ node, canUnlink, onBlur, onChange, onClose, onUnlink }: Props) {
+export function LinkMenu({
```
[the remark this answers is not in the document]
Comment (OskarGasiorowski) [reply packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/LinkMenu.tsx:21]: done

File: packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/LinkMenu.tsx (hunk 1 above)
Comment (OskarGasiorowski) [reply packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/LinkMenu.tsx]: done

**Review by aspirisen — APPROVED** [review]

**Comment by aspirisen** [comment]
@OskarGasiorowski do we have appropriate PR for setting this new flag in Prezly app?

**Comment by OskarGasiorowski** [comment]
> 

Nope, because I had problems with this app <-> slate linking. I do it right now :)

**Comment by OskarGasiorowski** [comment]
@aspirisen Nope, because I had problems with this app <-> slate linking. I do it right now :)

**Comment by OskarGasiorowski** [comment]
@aspirisen it's here: https://github.com/prezly/prezly/pull/10877

**Pull request merged**